model/receiver.go

```diff
@@ -36,7 +36,12 @@ func CreateHandler(ctx context.Context, command *Command, eventStore EventReposi
 		logger.Debug(err)
 		return err
 	}
-
+	//Generate id for entity
+	payload, err = GenerateID(payload, entityFactory)
```

Comment:
this should have been instantiating the ContentEntity with the payload (maybe we need a construction FromSchemaAndBuilderAndPayload)